main.js

```diff
@@ -0,0 +1,35 @@
+var swiper = new Swiper('.swiper--preview-img', {
+  loop: true,
+  spaceBetween: 10,
+  slidesPerView: 4,
+  freeMode: true,
+  watchSlidesProgress: true,
+
+});
+
+var swiper = new Swiper('.swiper--view-img', {
+  loop: true,
+  spaceBetween: 10,
+  navigation: {
+    nextEl: ".swiper-button-next",
+    prevEl: ".swiper-button-prev",
+  },
+  thumbs: {
+    swiper: swiper,
+  }
+
+});
+
+function sumar() {
+  var inputElement = document.getElementById("product__add-card-number");
```

Comment:
Usa las clases `js-`. Evita lo mas que puedas el uso de ids
```suggestion
  const inputElement = document.querySelector('.js-product-add-card-number');
```